main.go

```diff
@@ -308,3 +323,22 @@ func main() {
 		os.Exit(1)
 	}
 }
+
+// This function get the option from command argument (tlsConfig), check the validity through k8sapiflag
+// and set the config for webhook server.
+// refer to https://pkg.go.dev/k8s.io/component-base/cli/flag
+func tlsConfigSetting(cfg *tls.Config, tlsOpt tlsConfig) {
+	// TLSVersion helper function returns the TLS Version ID for the version name passed.
+	tlsVersion, err := k8sapiflag.TLSVersion(tlsOpt.minVersion)
+	if err != nil {
+		setupLog.Error(err, "TLS version invalid")
```

Comment:
I think we should be exiting on failure to honor this setting. The same applies to the cipher suites.

Reply:
Also it would be good to handle the interpreting of the tls options in main up front, and just pass the values into this function. That would keep any argument handling logic in a single place, and make sure it happens right away on startup similar to other argument parsing failures.

Reply:
Thank for highlighting on exiting on failure. For second point, my opinion is that when NewManager is called, it will callback on setting the TLS config, if we exit with failure, the controller will not able to start anyway. 
May b u could show me how u organize the argument handling logic and i could refer to it. Hehe..  I am not the expert in this domain and need your help to guide me.